framework/src/postprocessors/SideIntegralVariablePostprocessor.C

```diff
@@ -8,9 +8,13 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
 #include "SideIntegralVariablePostprocessor.h"
+#include "MathFVUtils.h"
 
 #include "metaphysicl/raw_type.h"
 
+using namespace Moose;
+using namespace FV;
```

Comment:
Is this bleeding into other .C files with unity build (as everything will be crammed into the same translation unit)?

Reply:
yes

Reply:
removed

Reply:
Using is scoped, so we could use it at the function level...

Reply:
I don't think you can scope a namespace at the class level, but I could be wrong. Anyway I think it's just as nice sometimes to have the explicit `Moose::FV::` scoping ... helps developers instantly grok a bit where a function is coming from